client/src/components/internship/EditInternship.vue

```diff
@@ -7,58 +8,56 @@
           <div class="col">
             <label for="startDate">
               {{ $t("internship.form.startDate") }}
-              <span v-if="startDate">({{ $t("internship.form.currently") }}: {{startDate}})</span>
             </label>
-            <input v-model="newStartDate"
+            <input v-model="startDate"
                    type="date"
                    id="startDate"
                    class="form-control"
-                   :placeholder="startDate"/>
+                  />
           </div>
           <div class="col">
-            <label for="startDate">
+            <label for="endDate">
               {{ $t("internship.form.endDate") }}
-              <span v-if="endDate">({{ $t("internship.form.currently") }}: {{endDate}})</span>
             </label>
-            <input v-model="newEndDate"
+            <input v-model="endDate"
                    type="date"
                    id="endDate"
                    class="form-control"
-                   :placeholder="endDate"/>
+                   />
           </div>
         </div>
 
         <div class="row my-4">
           <div class="col">
             <label for="operationalArea">{{ $t('internship.form.operationalArea') }}</label>
-            <input v-model="newOperationalArea"
+            <input v-model="internship.operationalArea"
                    type="text"
                    class="form-control"
                    id="operationalArea"
-                   :placeholder="operationalArea ?? $t('internship.form.operationalArea')"/>
+                   :placeholder="internship.operationalArea ?? $t('internship.form.operationalArea')"/>
           </div>
           <div class="col">
             <label for="programmingLanguages">
               {{ $t('internship.form.programmingLanguages') }}
             </label>
-            <input v-model="newProgrammingLanguages"
+            <input v-model="internship.programmingLanguages"
                    type="text"
                    class="form-control"
                    id="programmingLanguages"
-                   :placeholder="programmingLanguages ?? $t('internship.form.programmingLanguages')"
+                   :placeholder="internship.programmingLanguages?.toString().split(',').join(', ') ?? $t('internship.form.programmingLanguages')"
```

Comment:
```suggestion
                   :placeholder="internship.programmingLanguages?.toString().replaceAll(',', ', ') ?? $t('internship.form.programmingLanguages')"
```
Find ich ein bisschen schöner, ist aber wahrscheinlich nur personal preference 😄